lib/google_drive/session.rb

```diff
@@ -418,6 +418,24 @@ def collection_by_url(url)
 
     alias folder_by_url collection_by_url
 
+    # Returns GoogleDrive::Collection with given +key+.
+    #
+    # e.g.
+    #   # https://drive.google.com/drive/u/2/folders/1rPPuzAew4tO3ORc88Vz1JscPCcnrX7-J
+    #   session.collection_by_key("1rPPuzAew4tO3ORc88Vz1JscPCcnrX7-J")
+    def collection_by_key(key)
```

Comment:
Can you use `id` instead of `key` e.g., collection_by_id? `key` is a legacy terminology from the old Spreadsheet API. So it shouldn't be used for things other than spreadsheets. Do not add `key` method to GoogleDrive::File for the same reason.

Reply:
Ah, thanks for the heads up.  I haven't worked with the Google API much before this, so I was just following the pattern I saw in the spreadsheets file.  I'll change it.